### PR TITLE
[16.0][MIG] mgmtsystem_action_efficacy: Migration to 16.0

### PR DESCRIPTION
**Pull request by FernandoRomera**

### CODE REVIEW

**Review by max3903 — APPROVED** [review]

File: mgmtsystem_action_efficacy/models/mgmtsystem_action.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    efficacy_user_id = fields.Many2one(
+        "res.users",
+        "Inspector",
+        track_visibility=True,
+    )
+    # notes on the efficacy
+    efficacy_description = fields.Text("Notes")
+
+    @api.onchange("efficacy_value")
+    def _onchange_efficacy_value(self):
```
Comment (ivantodorovich) [inline mgmtsystem_action_efficacy/models/mgmtsystem_action.py:33]: Not specific to the migration, but rather to the original dev.
This should be in a `constraint`, not an `onchange`.

**Review by ivantodorovich — APPROVED** [review]

**Review by AntoniRomera — APPROVED** [review]

File: mgmtsystem_action_efficacy/models/mgmtsystem_action.py (hunk 1 above)
Comment (FernandoRomera) [reply mgmtsystem_action_efficacy/models/mgmtsystem_action.py:33]: Once approved, I will make a PR for it.

**Comment by FernandoRomera** [comment]
@max3903 
This is the only one left, for today. Can you merge this?

**Pull request merged**